crates/macro/src/lib.rs

```diff
@@ -308,6 +308,7 @@ impl Config {
             exports,
             ownership: self.ownership,
             runtime_path: Some("cargo_component_bindings::rt".to_string()),
+            bitflags_path: Some("cargo_component_bindings::bitflags".to_string()),
```

Comment:
This is pre-existing, but should these names be fully qualified with `::` at the start, eg. `::cargo_component_bindings::rt`?

Reply:
Definitely couldn't hurt, I'll fix.